docs/getting_started_guide/ISO_Install/SIMP_Server_Installation.rst

```diff
@@ -7,14 +7,32 @@ the SIMP server using the SIMP Utility, ``simp``.
 System Requirements
 -------------------
 
-SIMP scales well, but how much depends on a number of factors, including the
-number of nodes, the processor speed, the total memory, and the complexity of
-the manifests. The following are the minimal system requirements for the
-SIMP server:
-
--  :term:`Central Processing Unit` (CPU) : 2 Cores
--  :term:`Random Access Memory` (RAM) : 2.2 GB
--  :term:`Hard Disk Drive` (HDD) : 50 GB
+The scalability of SIMP correlates to the scalability of Puppet.  From the
+`Puppet tuning guide <https://docs.puppet.com/puppetserver/latest/tuning_guide.html>`_,
+a number of factors contribute to scalability, including:
+
+* Speed and quantity of available hardware
+* Number of nodes, and frequency of check-in
+* Number of modules in your module path
+* Amount of hieradata
+
+While there are no official
+`hardware requirements <https://docs.puppet.com/puppet/4.8/system_requirements.html>`_,
+Puppet recommends the following:
+
+* ``2`` CPUs and ``1 GB`` of RAM, at a minimum
```

Comment:
I didn't think puppetserver would start in systems with less that 3.5GB of RAM? Or is this for a client?

Reply:
That was for an agent, woops